src/js/components/stock-movement-wizard/modals/CombinedShipmentItemsModal.jsx

```diff
@@ -86,10 +86,14 @@ const FIELDS = {
         },
         getDynamicAttr: ({ values, rowIndex }) => {
           const orderItem = values.orderItems[rowIndex];
+          const productSupplierName = orderItem?.productSupplierName
+            ? `(source: ${orderItem?.productSupplierName})`
+            : null;
```

Comment:
Maybe it can be moved to a separate function for getting productSupplierName? I can see that it is used more than once